source/structured-cloneable.d.ts

```diff
@@ -11,37 +11,46 @@ type StructuredCloneablePrimitive =
 	| Number
 	| String;
 
+/** Grab the specified types from their global constructors, if they're declared */
+type ExtractFromGlobalConstructors<Name extends string> =
```

Comment:
This would be useful as an exported type. If you're not interested in doing that, at least put it in internal.d.ts, so we don't forget to eventually export it.